bats/tests/profile/invalid-locked-k8s-version.bats

```diff
@@ -0,0 +1,52 @@
+load '../helpers/load'
+
+local_setup() {
+    RD_USE_PROFILE=true
+}
+
+@test 'initial factory reset' {
+    factory_reset
+}
+
+@test 'create profile' {
+    PROFILE_TYPE=$PROFILE_LOCKED
+    RD_USE_PROFILE=true
```

Comment:
This is redundant; it has already been set in `local_setup()`. It is also only relevant for `start_container_engine()` (called by `start_application()`), so has no effect in this test anyways.